rb/lib/selenium/webdriver/common/logger.rb

```diff
@@ -38,20 +38,28 @@ class Logger
 
       def_delegators :@logger,
                      :close,
-                     :debug, :debug?,
-                     :info, :info?,
+                     :debug?,
+                     :info?,
                      :warn?,
                      :error, :error?,
                      :fatal, :fatal?,
-                     :level, :level=
+                     :level
 
       #
       # @param [String] progname Allow child projects to use Selenium's Logger pattern
       #
-      def initialize(progname = 'Selenium', ignored: nil)
+      def initialize(progname = 'Selenium', ignored: nil, select: nil, reject: nil)
         @logger = create_logger(progname)
-        @ignored = Array(ignored)
+        @reject = Array(Array(reject) + Array(ignored))
```

Comment:
I suggest you use sets for `@reject/@select`. This way you would have cleaner APIs (https://ruby-doc.org/stdlib-2.7.1/libdoc/set/rdoc/Set.html) when checking ids and won't have to worry about uniqueness and order.

Reply:
I guess I would prefer to use Arrays because there isn't exactly a "problem" with duplication, and Ruby makes arrays so much easier/flexible with splats, etc.